sources/packages/backend/apps/workers/src/services/student-assessment/student-assessment.model.ts

```diff
@@ -0,0 +1,5 @@
+import { StudentAssessment } from "@sims/sims-db";
+
+export class StudentAssessmentDetail extends StudentAssessment {
```

Comment:
A business comment would be recommended to explain that even with reassessments the original assessment is the one selected. If the business context was already added somewhere else please disregard this comment.